### PR TITLE
Add custom metrics for FederatedHPA

### PR DESCRIPTION
**Pull request by Poor12**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add custom metrics for FederatedHPA

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
None

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+---
+In Karmada, a FederatedHPA scales up/down the workload's replicas across multiple clusters, with the aim of automatically scaling the workload to match the demand.
+
+FederatedHPA not only supports resource metrics such as CPU and memory, but also supports custom metrics
+which may expand the use cases of FederatedHPA.
+
+This document walks you through an example of enabling FederatedHPA to automatically manage scale for a cross-cluster app with custom metrics.
+
+The walkthrough example will do as follows:  
+![federatedhpa-custom-metrics-demo](../resources/tutorials/custom_metrics.png)
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md:12]:
```suggestion

![federatedhpa-custom-metrics-demo](../resources/tutorials/custom_metrics.png)

```
Adding a blank line around the image can avoid cross-line on a big screen.

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+* One sample-deployment's pod exists in `member1` cluster.
+* The service is deployed in `member1` and `member2` cluster.
+* Request the multi-cluster service and trigger the pod's custom metrics(http_requests_total) increases.
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md:15]:
```suggestion
* Request the multi-cluster service and trigger an increase in the pod's custom metrics (http_requests_total).
```

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+* The replicas will be scaled up in `member1` and `member2` cluster.
+
+## Prerequisites
+
+### Karmada has been installed
+
+We can install Karmada by referring to [Quick Start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/local-up-karmada.sh` script which is also used to run our E2E cases.
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md:22]:
```suggestion
You can install Karmada by referring to [Quick Start](https://github.com/karmada-io/karmada#quick-start),
or directly run `hack/local-up-karmada.sh` script which is also used to run your E2E cases.
```

**Review by windsonsea — COMMENTED** [review]
Both `We` and `You` exist on a page, `You` is better in a tutorial IMO

File: docs/tutorials/autoscaling-with-custom-metrics.md [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+
+### prometheus and prometheus-adapter have been installed in member clusters
+
+You need to install `prometheus` and `prometheus-adapter` for member clusters to provider the custom metrics.
+You can install it by running the following in member clusters:
+```sh
+git clone https://github.com/prometheus-operator/kube-prometheus.git
+cd kube-prometheus
+kubectl apply --server-side -f manifests/setup
+kubectl wait \
```
Comment (jwcesign) [inline docs/tutorials/autoscaling-with-custom-metrics.md:83]: Can we use one specific version?

File: docs/userguide/autoscaling/federatedhpa.md [hunk 5]
```diff
@@ -35,3 +35,4 @@ If you configure FederatedHPA, you may also want to consider running a cluster-l
 
 For more information on FederatedHPA:
 * Read a [FederatedHPA tutorials](../../tutorials/autoscaling-with-federatedhpa.md) for FederatedHPA.
```
Comment (jwcesign) [inline docs/userguide/autoscaling/federatedhpa.md:37]: How about renaming this to `FederatedHPA tutorials with CPU/Memory utilization metrics`?

Comment (jwcesign) [reply docs/userguide/autoscaling/federatedhpa.md:37]: Sorry, I mean changing line 37 to this.

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**